### DISCUSSION TITLE
Description field

### DISCUSSION THREAD

**Discussion by onegrasshopper** [rsjaffe/MIDI2LR#605] [Ideas]
It would be nice to have a basic text editable description column/field in the gui and xml.

This would allow for writing/sorting of self written notes. 
One use case would be to write a short description of what the button or knob is on the device. It would then be easy to load the settings xml into Excel/etc and pretty print.
Another use case would be to write notes on any particular relative/resolution/etc cc settings. (Helpful since these are not saved with the named xml but in a general settings.xml written out by juce sdk in SaveControlsModel)

Thank you.

**Comment by rsjaffe** [comment]
Sounds interesting, but I'm not going to work on that right now. As noted in my answer to your other question, I welcome contributors to code. I may sound harsh stating I'm not going to work on that, but my higher priorities are the various glitches and bugs that prevent some people from using MIDI2LR at all. For example, a new one is starting to show up sporadically where it appears that the new Lightroom threading model (changed with version 11) is having trouble on some machines keeping up with MIDI2LR requests (e.g., #590 and #593). And since I can't reproduce those issues on my machine, it takes a long time and lots of experimentation to work them out.
